package.json

```diff
@@ -7,7 +7,9 @@
         "prepare": "husky install",
         "test": "ts-mocha src/tests/**.ts",
         "dev": "yarn && ts-node-dev --transpile-only --no-notify --exit-child src/index.ts",
-        "docker:run": "ts-node src/index.ts",
```

Comment:
Hey @ermish I like your script, but it made more sense to me to make `docker:run` connect to a docker command. So I called this one `prod`. Let me know your thoughts!

Reply:
I like the direction! What if we took it one step further and the prod script runs all the docker scripts together to spin up the whole thing locally? 

so we'd have something like 
`yarn dev` -> run  locally without docker
`yarn prod` -> run docker commands all together `yarn docker:build && yarn docker:run`

Reply:
We still need a name for the script that runs ts-node.... I'd like that to be `prod` or something similar

And I want to ensure we have scripts for running bare-metal.

So if you want another script, pick a name that is unclaimed

Reply:
'Serve' sounds too close to dev, some projects use it

Reply:
`yarn staging`?